src/topology/algebra/module/locally_convex.lean

```diff
@@ -92,3 +92,61 @@ lemma locally_convex_space_iff_exists_convex_subset_zero :
 (locally_convex_space_iff_zero 𝕜 E).trans has_basis_self
 
 end module
+
+section lattice_ops
+
+variables {ι : Sort*} {𝕜 E F : Type*} [ordered_semiring 𝕜] [add_comm_monoid E]
+  [module 𝕜 E] [add_comm_monoid F] [module 𝕜 F] {ts : set (topological_space E)}
+  (h : ∀ t ∈ ts, @locally_convex_space 𝕜 E  _ _ _ t) {ts' : ι → topological_space E}
+  (h' : ∀ i, @locally_convex_space 𝕜 E  _ _ _ (ts' i)) {t₁ t₂ : topological_space E}
+  (h₁ : @locally_convex_space 𝕜 E _ _ _ t₁) (h₂ : @locally_convex_space 𝕜 E _ _ _ t₂)
+  {t : topological_space F} [locally_convex_space 𝕜 F] {f : E →ₗ[𝕜] F}
```

Comment:
You declare a bunch of explicit variables, then use each of them exactly once. Could you please move them to lemmas?